yt/frontends/ramses/data_structures.py

```diff
@@ -244,13 +262,15 @@ def _read_particle_header(self):
                 ("particle_identifier", "i"),
                 ("particle_refinement_level", "I")]
 
-        if self.ds._extra_particle_fields is not None:
-            particle_fields += self.ds._extra_particle_fields
+            if self.ds._extra_particle_fields is not None:
+                particle_fields += self.ds._extra_particle_fields
+
+            ptype = 'io'
```

Comment:
`ptype = 'io'` is the case for both variants. Could you move it out of `if/else` statement?